remix_01.py

```diff
@@ -51,13 +59,17 @@ def filter_json(id):
             elif child.has_key('pool_item_id'):
                 remix_children.append(child['pool_item_id'])
             
+    # Create a new dict for this remix
```

Comment:
Instead of printing the data, we add the data as property to a new `remix` _dict_